locales/en/plugin__odf-console.json

```
@@ -194,6 +217,16 @@
   "Recovery": "Recovery",
   "Block and File service is unhealthy": "Block and File service is unhealthy",
   "Object service is unhealthy": "Object service is unhealthy",
+  "Delete BlockPool": "Delete BlockPool",
+  "<0><0>{{poolName}}</0> cannot be deleted. When a pool is bounded to PVC it cannot be deleted. Please detach all the resources from StorageClass(es):</0>": "<0><0>{{poolName}}</0> cannot be deleted. When a pool is bounded to PVC it cannot be deleted. Please detach all the resources from StorageClass(es):</0>",
+  "<0>Deleting <1>{{poolName}}</1> will remove all the saved data of this pool. Are you sure want to delete?</0>": "<0>Deleting <1>{{poolName}}</1> will remove all the saved data of this pool. Are you sure want to delete?</0>",
+  "BlockPool Delete Modal": "BlockPool Delete Modal",
```

```suggestion
  "BlockPool Delete Modal": "BlockPool delete modal",
```
shouldn't it is sentence case ? Just confirming...

---

Taking it as is.